ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -108,4 +109,11 @@ def _process_step_infos(self, step_infos: List[EnvironmentStep]) -> int:
                         name_behavior_id, ActionInfo.empty()
                     ),
                 )
+
+                # In order to prevent conflicts between multiple environments,
```

Comment:
This would be helpful documentation in the C# class too.

Reply:
I am rather hesitant on this. Are the stats only usable when training? Is it not possible for users to get the C# stats when using the LL-API? If it is possible to use it with LL-API, then C# should not assume how the stats are consumed (at least not without a warning of type : "When using ml-agents training ™️, the stats will...")

Reply:
I moved it to a comment on StatAggregationMethod.MostRecent:
> To avoid conflicts between multiple environments, the ML Agents environment will only
> keep stats from worker index 0.